Research Papers/Shallow learning/README.md

```diff
@@ -289,22 +333,36 @@ This paper investigates a relatively new direction in Multiagent Reinforcement L
 
 <details> <summary> <a href="https://link.springer.com/content/pdf/10.1007/3-540-45356-3.pdf"> A Game-Theoretic Approach to the Simple Coevolutionary Algorithm </a>by Sevan G. Ficici, Jordan B. Pollack. LNCS, 2000. <a href="link">  </a> </summary> The fundamental distinction between ordinary evolutionary algorithms (EA) and co-evolutionary algorithms lies in the interaction between coevolving entities. We believe that this property is essentially game-theoretic in nature. Using game theory, we describe extensions that allow familiar mixing-matrix and Markov-chain models of EAs to address coevolutionary algorithm dynamics. We then employ concepts from evolutionary game theory to examine design aspects of conventional coevolutionary algorithms that are poorly understood. <br> - </details>
 
+<details> <summary> <a href="https://www.aaai.org/Papers/AAAI/2000/AAAI00-031.pdf?ref=https://githubhelp.com"> On pruning techniques for multi-player games </a>by N. Sturtevant and R. Korf. In Proceedings of National Conference on Artificial Intelligence (AAAI), pages 201–207, 2000. <a href="link">  </a> </summary> Max-n (Luckhardt and Irani, 1986) is the extension of the minimax backup rule to multi-player games. We have shown that only a limited version of alpha-beta pruning, shallow pruning, can be applied to a max-n search tree. We extend this work by calculating the exact bounds needed to use this pruning technique. In addition, we show that branch-and-bound pruning, using a monotonic heuristic, has the same limitations as alpha-beta pruning in a max-n tree. We present a hybrid of these algorithms, alpha-beta branch-and-bound pruning, which combines a monotonic heuristic and backed-up values to prune even more effectively. We also briefly discuss the reduction of a n-player game to a "paranoid" 2-player game. In Sergeant Major, a 3-player card game, we averaged node expansions over 200 height 15 trees. Shallow pruning and branch-and-bound each reduced node expansions by a factor of about 100. Alpha-beta branch-and-bound reduced the expansions by an additional factor of 19. The 2-player reduction was a factor of 3 better than alpha-beta branchand-bound. Using heuristic bounds in the 2-player reduction reduced node expansions another factor of 12. <br> - </details>
```

Comment:
Not sure this should be here? Does not seem to be related to evolutionary strategies?